integration/feature/init/src/MillInitSbtTests.scala

```diff
@@ -0,0 +1,332 @@
+package mill.integration
+
+import mill.constants.Util
+import mill.integration.testMillInit
+import utest.*
+
+private def bumpSbtTo1107(workspacePath: os.Path) =
+  // bump sbt version to resolve compatibility issues with lower sbt versions and higher JDK versions
+  os.write.over(workspacePath / "project" / "build.properties", "sbt.version = 1.10.7")
+
+// relatively small libraries
+
+private val scalaPlatforms = Seq("js", "jvm", "native")
+
+object MillInitSbtLibraryExampleTests extends BuildGenTestSuite {
+  def tests: Tests = Tests {
+    /*
+    - 21 KB
+    - sbt 1.5.2
+     */
+    val url = "https://github.com/scalacenter/library-example/archive/refs/tags/v1.0.1.zip"
+
+    test - integrationTest(url)(
+      _.testMillInit(
+        expectedCompileTasks = Some(SplitResolvedTasks(Seq("compile"), Seq())),
+        expectedTestTasks = None // scalaprops not supported in `TestModule`
+      )
+    )
+  }
+}
+
+object MillInitSbtScalaCsv200Tests extends BuildGenTestSuite {
+  def tests: Tests = Tests {
+    /*
+    - 34 KB
+    - originally sbt 1.10.0
+     */
+    val url = "https://github.com/tototoshi/scala-csv/archive/refs/tags/2.0.0.zip"
+
+    test - integrationTest(url) { tester =>
+      bumpSbtTo1107(tester.workspacePath)
+
+      // Cross builds are not supported yet.
+      tester.testMillInit(
+        expectedCompileTasks = Some(SplitResolvedTasks(Seq(), Seq("compile", "test.compile"))),
+        expectedTestTasks = Some(SplitResolvedTasks(Seq(), Seq("test")))
+      )
+    }
+  }
+}
+
+object MillInitSbtScalaCsv136Tests extends BuildGenTestSuite {
+  def tests: Tests = Tests {
+    /*
+    - 28 KB
+    - originally sbt 1.2.8
+     */
+    val url = "https://github.com/tototoshi/scala-csv/archive/refs/tags/1.3.6.zip"
+
+    test - integrationTest(url) { tester =>
+      bumpSbtTo1107(tester.workspacePath)
+
+      tester.testMillInit(
+        expectedCompileTasks = Some(SplitResolvedTasks(
+          Seq("compile", "test.compile"),
+          Seq.empty
+        )),
+        expectedTestTasks = Some(SplitResolvedTasks(
+          Seq(),
+          /*
+          Paths relative to the workspace are used in the test sources such as `new File("src/test/resources/simple.csv")`
+          and they seem to cause the test to fail with Mill:
+          ```text
+          java.io.FileNotFoundException: src/test/resources/simple.csv (No such file or directory)
+          ```
+           */
```

Comment:
Could we try generating `def testSandboxWorkingDir = false` to see if that fixes this issue? Setting that should make the test process run in the repo root IIRC rather than in some sandbox subfolder in `out/`

Reply:
Alternatively we could try a different repo for the "small library" test, e.g. https://github.com/scopt/scopt or https://github.com/scallop/scallop or https://github.com/typelevel/jawn

Reply:
I added some code to directly write `def testSandboxWorkingDir = false` to the converted `build.mill` file and it works. Shall I update `BuildGenUtil` to render this line for the test modules automatically?

Reply:
Yes let's generate that by default. That flag is meant to ease in migration, so this is exactly what it's meant for